src/spec-utils/log.ts

```diff
@@ -111,6 +111,10 @@ export function createCombinedLog(logs: LogHandler[], header?: string): LogHandl
 					text: header!,
 				}));
 			}
+
+			if (e.type !== 'progress') {
```

Comment:
Does `stepDetail` for `'progress'` events need the same treatment? Not sure anymore what could be in there. /cc @samruddhikhandale

Reply:
Added masking for stepDetail as well, since we also write progress events into json logs